### PR TITLE
Scala 2.13.0-M4

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: scalikejdbc-core/src/main/scala/scalikejdbc/OneToManySQL.scala [hunk 1]
```diff
@@ -17,13 +17,13 @@ private[scalikejdbc] trait OneToManyExtractor[A, B, E <: WithExtractor, Z]
     if (oneToMany.contains(o)) {
       extractTo(rs).map(many => oneToMany += (o -> (oneToMany.apply(o) :+ many))).getOrElse(oneToMany)
     } else {
-      oneToMany += (o -> extractTo(rs).map(many => Vector(many)).getOrElse(Nil))
+      oneToMany += (o -> extractTo(rs).map(many => Vector(many): Seq[B]).getOrElse(Nil))
```
Comment (xuwei-k) [inline scalikejdbc-core/src/main/scala/scalikejdbc/OneToManySQL.scala]: https://github.com/scala/bug/issues/10907

File: scalikejdbc-core/src/main/scala/scalikejdbc/interpolation/SQLSyntax.scala [hunk 2]
```diff
@@ -255,7 +255,8 @@ object SQLSyntax {
     }
     apply(value, parameters)
   }
-  def csv(parts: SQLSyntax*): SQLSyntax = join(parts, sqls",", false)
+  def csv(parts: SQLSyntax*): SQLSyntax = commaSeparated(parts)
+  private[scalikejdbc] def commaSeparated(parts: scala.collection.Seq[SQLSyntax]): SQLSyntax = join(parts, sqls",", false)
```
Comment (seratch) [inline scalikejdbc-core/src/main/scala/scalikejdbc/interpolation/SQLSyntax.scala]: I understand we have to have another name for this. I prefer `toCSV` than `commaSeparated`.

Comment (xuwei-k) [reply scalikejdbc-core/src/main/scala/scalikejdbc/interpolation/SQLSyntax.scala]: renamed and `push -f`

**Comment by seratch** [comment]
Thanks, I believe that we can merge this now.

**Pull request merged**